scimono-server/src/main/java/com/sap/scimono/api/QueryParamParser.java

```diff
@@ -0,0 +1,24 @@
+
+package com.sap.scimono.api;
+
+import com.sap.scimono.exception.InvalidInputException;
+
+public class QueryParamParser {
```

Comment:
Since you are not processing all the query params here (e.g. filter is also query param but it is not here) you must name this class otherwise. What about PagingParamsParser.
Also, If you don't need to keep a state. (using only static methods), add private default constructor as well